package.json

```
@@ -2,7 +2,7 @@
   "name": "overwatch-workshop",
   "private": true,
   "engines": {
-    "node": "^14.0.0"
+    "node": "^18.12.0"
```

I think we could be more lenient here

```suggestion
    "node": ">18"
```